src/WebJobs.Script/Diagnostics/Extensions/LoggerExtension.cs

```diff
@@ -121,15 +121,15 @@ internal static class LoggerExtension
                 new EventId(325, nameof(ScriptStartUpLoadedExtension)),
                 "Loaded extension '{startupExtensionName}' ({startupExtensionVersion})");
 
-        private static readonly Action<ILogger, Exception> _functionMetadataProviderReadingMetadata =
-            LoggerMessage.Define(LogLevel.Information,
+        private static readonly Action<ILogger, string, Exception> _functionMetadataProviderReadingMetadata =
+            LoggerMessage.Define<string>(LogLevel.Information,
                 new EventId(326, nameof(FunctionMetadataManagerLoadingFunctionsMetadata)),
-                "Reading functions metadata");
+                "Reading functions metadata from {source}");
 
-        private static readonly Action<ILogger, int, Exception> _functionMetadataProviderFunctionsFound =
-            LoggerMessage.Define<int>(LogLevel.Information,
+        private static readonly Action<ILogger, int, string, Exception> _functionMetadataProviderFunctionsFound =
```

Comment:
Similarly I think we should rename this to _functionsReturnedByProvider and FunctionsReturnedByProvider respectively